docs/derek/examples/_html.ejs

```diff
@@ -1,41 +1,636 @@
-<h2>Examples</h2>
-<p>Below is a list of examples to help you get started building your web pages.</p>
-<div class="row text-center">
-  <div class="col-md-4">
-    <a href="landing-page.html">
-      <img src="imgs/derek/examples/landing-page.png" class="img-thumbnail" />
-    </a>
-    <p>Landing Page</p>
-  </div>
-  <div class="col-md-4">
-    <a href="javascript:">
-      <img src="http://placehold.it/240x300" class="img-thumbnail" />
-    </a>
-    <p>Some Example</p>
-  </div>
-  <div class="col-md-4">
-    <a href="javascript:">
-      <img src="http://placehold.it/240x300" class="img-thumbnail" />
-    </a>
-    <p>Some Example</p>
-  </div>
-  <div class="col-md-4">
-    <a href="javascript:">
-      <img src="http://placehold.it/240x300" class="img-thumbnail" />
-    </a>
-    <p>Some Example</p>
-  </div>
-  <div class="col-md-4">
-    <a href="javascript:">
-      <img src="http://placehold.it/240x300" class="img-thumbnail" />
-    </a>
-    <p>Some Example</p>
-  </div>
-  <div class="col-md-4">
-    <a href="javascript:">
-      <img src="http://placehold.it/240x300" class="img-thumbnail" />
-    </a>
-    <p>Some Example</p>
+
+
+
+<!-- main content wrap -->
```

Comment:
Yo the `docs` directory DOES NOT EXIST

Please rebase your work on the latest master